CHANGELOG.md

```diff
@@ -1,24 +1,22 @@
 ## vNext (TBD)
+NOTE: Since the file format of the Realm aux. files has been changes, and it is required to use Realm Studio v14.0.0 to open Realm files produced by this release.
```

Comment:
```suggestion
NOTE: Since the file format of the Realm files has been changed, it is required to use Realm Studio v14.0.0 to open Realm files produced by this release.
```

Reply:
Updated my suggestion